docs/7-vector-search/5-aws-sagemaker.mdx

```diff
@@ -0,0 +1,109 @@
+# 🦸 Using Amazon SageMaker
+
+[Amazon SageMaker](https://aws.amazon.com/pm/sagemaker/) is a cloud based machine-learning platform that enables developers to build, train, and deploy machine learning (ML) models for any use case with fully managed infrastructure, tools, and workflows.
+It also enables developers to deploy ML models on embedded systems and edge-devices. SageMaker was launched in November 2017 and has been improved ever since, especially since the increased popularity and demand of ML and AI systems in recent years.
+
+To make it easier to get started, Amazon SageMaker JumpStart provides a set of solutions for the most common use cases that can be deployed readily with just a few clicks.
+
+Amazon SageMaker is available for free, for 2 months, as part of the AWS Free Tier program. Users can get access to 250 hours per month of `ml.t3.medium` notebooks usage with the Free Tier.
+
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/0-pricing.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
```

Comment:
I am not a big fan of screenshots since they need to be redone every time the UI changes. Especially when it can be replaced with some text. In this care, I would completely remove the pricing part, and just add a link to where they can find the information.